conf.d/Makefile.am

```diff
@@ -6,4 +6,20 @@ MAINTAINERCLEANFILES= $(srcdir)/Makefile.in
 dist_config_DATA = \
 	apps_groups.conf \
 	charts.d.conf \
+	python.d.conf \
 	$(NULL)
+
+chartsconfigdir=$(configdir)/charts.d
+dist_chartsconfig_DATA = \
+	$(NULL)
+
+nodeconfigdir=$(configdir)/node.d
+dist_nodeconfig_DATA = \
+	$(NULL)
+
+pythonconfigdir=$(configdir)/python.d
+dist_pythonconfig_DATA = \
+	python.d/example.conf
```

Comment:
I've installed it with `netdata-installer.sh` on clean ubuntu 16.04 and `python.d/example.conf` was where it should. So I think it is ok.